source/Halibut.Tests/Timeouts/TimeoutsApplyDuringHandShake.cs

```diff
@@ -38,7 +38,7 @@ public class TimeoutsApplyDuringHandShake : BaseTest
             var dataTransferObserverDoNothing = new DataTransferObserverBuilder().Build();
 
             var halibutTimeoutsAndLimits = new HalibutTimeoutsAndLimitsForTestsBuilder().Build().WithAllTcpTimeoutsTo(TimeSpan.FromMinutes(20));
-            halibutTimeoutsAndLimits.TcpClientAuthenticationAndIdentificationTimeouts = new SendReceiveTimeout(TimeSpan.FromSeconds(6), TimeSpan.FromSeconds(6));
+            halibutTimeoutsAndLimits.TcpClientTimeout = new SendReceiveTimeout(TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(10));
```

Comment:
Locally I would sometimes see that the "valid" request done at the very end of the test would fail because the machine was so busy it could not get a write done in `6s`.